editor/editor_file_system.cpp

```diff
@@ -1184,7 +1255,10 @@ void EditorFileSystem::scan_changes() {
 		scan_total = 0;
 		Thread::Settings s;
 		s.priority = Thread::PRIORITY_LOW;
+		// Accrues changes
 		thread_sources.start(_thread_func_sources, this, s);
+		// Because of multithreading, changes are applied in a check that occurs in the process loop.
+		// If the loop detects the multithread is done, then it applies the changes. See notification Process for more.
```

Comment:
Use exact name to make searching easier:

```suggestion
		// If the loop detects the multithread is done, then it applies the changes. See NOTIFICATION_PROCESS for more.
```